ci/scripts/run-english-breakfast-verilator-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Only a single test is supported on English Breakfast (EB).
+# Currently, Bazel cannot build the EB Verilator model, so we only build the test with Bazel and then use opentitantool directly
+
+set -e
+
+. util/build_consts.sh
+
+# Cleaning is necessary for the find commands below to work correctly
+ci/bazelisk.sh clean
+
+# Build Verilator
+ci/scripts/build-chip-verilator.sh englishbreakfast
+
+# Build the modified EB software.
+./hw/top_englishbreakfast/util/prepare_sw.py --bazel -b
+
+# Build some other dependencies.
+ci/bazelisk.sh build //sw/host/opentitantool //hw/ip/otp_ctrl/data:rma_image_verilator
+
+# Run the one test.
+bazel-bin/sw/host/opentitantool/opentitantool \
```

Comment:
If this can be run within bazel, please use "`bazel run`, if it can't (Which I think is the case) can you add a comment to that effect

Reply:
Added a comment